### PR TITLE
Gui line

### PR DESCRIPTION
**Pull request by SvenFrankson**
When called on Line controls, MoveToVector3 defaults to moving only one end of the line. This PR adds an optional boolean to move the other end of the line.

### CODE REVIEW

File: dist/preview release/what's new.md.orig [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- (Viewer) Scene Optimizer intergrated in viewer. ([RaananW](https://github.com/RaananW))
+- (Viewer) The viewer supports custom shaders in the configuration. ([RaananW](https://github.com/RaananW))
+- Documented PostProcessRenderEffect, DefaultRenderingPipeline, BlurPostProcess, DepthOfFieldEffect, PostProcess, PostProcessManager classes ([trevordev](https://github.com/trevordev))
+<<<<<<< HEAD
+- SPS internal storage of each solid particle rotation matrix ([jbousquie](https://github.com/jbousquie))
+- GUI.Line can have its world position set from one end or the other ([SvenFrankson](https://github.com/SvenFrankson))
+=======
+- SPS internal storage of each solid particle rotation matrix ([jbousquie](https://github.com/jbousquie)) 
+- (Viewer) Introducing the viewer labs - testing new features. ([RaananW](https://github.com/RaananW))
+>>>>>>> a2fb32783a63f248434630a7d32d0a7f647fb946
```
Comment (deltakosh) [inline dist/preview release/what's new.md.orig]: Seems wrong :)

**Comment by SvenFrankson** [comment]
Guess I just broke the project. My pleasure !

**Comment by deltakosh** [comment]
Still conflict and head lines :(

**Comment by deltakosh** [comment]
Still not good

File: dist/preview release/what's new.md.orig (hunk 1 above)
Comment (deltakosh) [reply dist/preview release/what's new.md.orig]: There is still this issue in this file. Please remove this.orig file from the pr

**Comment by deltakosh** [comment]
Could you remove the file ending with.orig?

**Pull request merged**